Implementation/59251 fully remove feature flag for primerized activity tab

jjabari-op opened the pull request:
[Ticket](https://community.openproject.org/work_packages/59251/activity)

jjabari-op committed:
[#59040] Prepare pull request to remove primerised work packages feature flag

https://community.openproject.org/work_packages/59040

jjabari-op committed:
WIP remove feature flag

jjabari-op committed:
Merge branch 'dev' into task/59040-prepare-pull-request-to-remove-primerised-work-packages-feature-flag

jjabari-op committed:
fixed custom_actions_spec

jjabari-op committed:
removed obsolete specs

jjabari-op committed:
fixed gitlab activity spec

jjabari-op committed:
fixed notification_center_spec via adjusted expect_wp_has_been_created_activity method

akabiru commented on config/initializers/feature_decisions.rb:
```diff
@@ -39,7 +39,6 @@
 #     OpenProject::FeatureDecisions.add :some_flag
 #   end
 
-OpenProject::FeatureDecisions.add :primerized_work_package_activities
```
Remember to revert https://github.com/opf/openproject/pull/17192 as well 🧹 ✨

jjabari-op committed:
fixed working_days_spec

jjabari-op committed:
fixed edit_work_package_spec

jjabari-op committed:
fixed work_package_workflow_form_spec

jjabari-op committed:
fixed scheduling_mode_spec

jjabari-op committed:
fixed copy_spec

jjabari-op committed:
cleanup

jjabari-op committed:
Merge branch 'dev' into implementation/59251-fully-remove-feature-flag-for-primerized-activity-tab

jjabari-op committed:
removed old revision spec as it will be substituted by a new spec covering the basic revision support in the primerized activity tab via Bug/59374

jjabari-op committed:
fixed mentions spec

jjabari-op committed:
fixed github module specs

jjabari-op committed:
fixed gitlab specs

jjabari-op committed:
fixed work_package_activity_spec

jjabari-op committed:
fixed shared context

jjabari-op committed:
subject editor spec cleanup

jjabari-op committed:
fixed notification bubble spec

jjabari-op committed:
updated notification center spec with new center behavior around auto-update

jjabari-op committed:
fixed and cleaned up activity_comments_spec markdown specs

jjabari-op committed:
Merge branch 'dev' into implementation/59251-fully-remove-feature-flag-for-primerized-activity-tab

jjabari-op committed:
fixed subject editor spec

jjabari-op committed:
fixing and marking activity_page_navigation_spec as pending

jjabari-op committed:
mark github/gitlab clipboard specs as pending

jjabari-op committed:
fix edit_work_package_spec

jjabari-op committed:
fixed share access spec

jjabari-op committed:
fixing attachement and edit_work_package specs, cleanup abstract_work_package page util

jjabari-op committed:
fixing rubocop issue

jjabari-op committed:
removing temp feature flag hack from ##17192

jjabari-op committed:
WIP frontend code removal

jjabari-op committed:
finalized frontend code removal

jjabari-op committed:
trying to fix flaky spec, which passes locally but not on CI

jjabari-op committed:
switch back to selenium

jjabari-op committed:
mark clipboard based specs as pending

jjabari-op commented:
@akabiru I've finalized the cleanup! What I've done:

- Adjusted/Fixed all specs which targeted the old activity tab
- Removed all frontend code, which is not required anymore
- Marked the github/gitlab specs, which rely on clipboard access as pending. I was locally not able to get the clipboard thing working in the tests as I run into permission issues in the headless browser. I prefer to tackle them in a separate PR and merge this one quickly before it gets stale

Happy to get your final review on this one. Thank you!

Best, Jonas

akabiru reviewed (approved):
Nice one @jjabari-op - looks good from my end. Manual QA good as well. I've requested @oliverguenther to have a quick scan as well 👍🏾

oliverguenther reviewed (approved):
Nice work @jonasjabari 👏
